source/agora/common/Deserializer.d

```diff
@@ -66,6 +67,16 @@ public void deserializePart (ref Hash record, scope DeserializeDg dg)
     record = Hash(dg(Hash.sizeof));
 }
 
+/// Ditto
+public void deserializePart (ref TxType record, scope DeserializeDg dg)
```

Comment:
I would say we handle this business within `Transaction`. This module should not depend on `consensus` module

Reply:
I've remove the `Transaction`  It's make not depend on consensus module.